pages/apply-dev.html

```diff
@@ -33,7 +33,8 @@ <h3 class="info-header">What You&rsquo;ll be doing</h3>
 
   <section class="info">
     <h3 class="info-header">Applications</h3>
-    <p class="info-secondary">Applications for our 2021 Full-Stack Developer Apprenticeship will open in fall of 2020. Be sure to <a href="https://twitter.com/hearsparkbox">follow us on Twitter</a> and check <a href="https://seesparkbox.com/foundry/">The Foundry</a> for updates.</a
+     <p class="info-secondary">If you feel like you’d be a great fit for our remote full-stack apprenticeship, we encourage you to apply.</p>	    
```

Comment:
Are you okay with me encoding this apostrophe (and scooching it back to align with the sibling elements)?
```suggestion
    <p class="info-secondary">If you feel like you’d be a great fit for our remote full-stack apprenticeship, we encourage you to apply.</p>	    
```